app/NtscApp.py

```diff
@@ -133,7 +133,7 @@ def __init__(self):
         self.exportImportConfigButton.clicked.connect(self.export_import_config)
 
         # TEMP HIDE WHILE FFPROBE ISSUE ISNT FIX
-        #self.ProcessAudioCheckBox.hide()
+        self.ProcessAudioCheckBox.hide()
```

Comment:
Hi
Sound will not work in redistributable version

Do you use ffprobe in processing
This is a separate binary file that weighs about 60mb
It's not included in the build right now.

In the last review, I wrote that, if possible, you need to find a workaround to find out the data without ffprobe, so as not to include a 60mb binary in the bundle just for the sake of it.
However, if there are no workarounds, then in order for ffprobe to get into the assembly, it is necessary not only to change the download link.
It is necessary to provide for the correct unpacking (because now there was a fix for a different type of archive and the directory structure inside it)
And add the ffprobe binary to the build spec